app/services/search/book_versions/i1/figure_document.rb

```diff
@@ -0,0 +1,20 @@
+module Search::BookVersions::I1
+  class FigureDocument < PageElementDocument
+
+    def initialize(element:, page_position:, page_id:)
```

Comment:
maybe "element" could be more specifically named?   openstax_element, xpath_element, or something...to make it clearer.